### PR TITLE
Update YOURLS to v1.7.3

### PR DESCRIPTION
**Pull request by LeoColomb**

### CODE REVIEW

File: library/yourls [hunk 1]
```diff
@@ -4,17 +4,17 @@ Maintainers: YOURLS <yourls@yourls.org> (@YOURLS),
              Léo Colombaro <git@colombaro.fr> (@LeoColomb)
 GitRepo: https://github.com/YOURLS/docker-yourls.git
 
-Tags: 1.7.2-apache, 1.7-apache, 1-apache, apache, 1.7.2, 1.7, 1, latest
```
Comment (yosifkit) [inline library/yourls]: Did you want to lose all these tag aliases, here and below? It looks like you just need a bump in [`generate-stackbrew-library.sh`](https://github.com/YOURLS/docker-yourls/blob/652a3971b92ab2e0bfa96ec78342f8757071dbf1/generate-stackbrew-library.sh#L5).

Comment (LeoColomb) [reply library/yourls]: Oops, good catch! 😇

Comment (LeoColomb) [reply library/yourls]: Fixed, thanks!

**Comment by yosifkit** [comment]
<details>
<summary>Diff:</summary>

```diff
diff --git a/_bashbrew-list b/_bashbrew-list
index 33e5ade..e5a840f 100644
--- a/_bashbrew-list
+++ b/_bashbrew-list
@@ -6,10 +6,10 @@ yourls:1.7
 yourls:1.7-apache
 yourls:1.7-fpm
 yourls:1.7-fpm-alpine
-yourls:1.7.2
-yourls:1.7.2-apache
-yourls:1.7.2-fpm
-yourls:1.7.2-fpm-alpine
+yourls:1.7.3
+yourls:1.7.3-apache
+yourls:1.7.3-fpm
+yourls:1.7.3-fpm-alpine
 yourls:apache
 yourls:fpm
 yourls:fpm-alpine
diff --git a/yourls_fpm-alpine/Dockerfile b/yourls_fpm-alpine/Dockerfile
index 9cce6e3..ddba60d 100644
--- a/yourls_fpm-alpine/Dockerfile
+++ b/yourls_fpm-alpine/Dockerfile
@@ -19,7 +19,7 @@ RUN apk add --no-cache bash
 
 VOLUME /var/www/html
 
-ENV YOURLS_VERSION 1.7.2
+ENV YOURLS_VERSION 1.7.3
 #ENV YOURLS_SHA256 %%SHA256%%
 
 RUN set -ex; \
diff --git a/yourls_fpm/Dockerfile b/yourls_fpm/Dockerfile
index efbd067..5321266 100644
--- a/yourls_fpm/Dockerfile
+++ b/yourls_fpm/Dockerfile
@@ -18,7 +18,7 @@ RUN { \
 
 VOLUME /var/www/html
 
-ENV YOURLS_VERSION 1.7.2
+ENV YOURLS_VERSION 1.7.3
 #ENV YOURLS_SHA256 %%SHA256%%
 
 RUN set -ex; \
diff --git a/yourls_latest/Dockerfile b/yourls_latest/Dockerfile
index 041d066..8cb987e 100644
--- a/yourls_latest/Dockerfile
+++ b/yourls_latest/Dockerfile
@@ -19,7 +19,7 @@ RUN a2enmod rewrite expires
 
 VOLUME /var/www/html
 
-ENV YOURLS_VERSION 1.7.2
+ENV YOURLS_VERSION 1.7.3
 #ENV YOURLS_SHA256 %%SHA256%%
 
 RUN set -ex; \
```

</details>

**Comment by yosifkit** [comment]
Build test of #5288; 447515f5; `amd64` (`yourls`):

```console

$ bashbrew build yourls:1.7.3-apache
Using bashbrew/cache:6f0d2d0229e210dff7fb45355ef6041c69ebd080a4db2bb22c6868b76fb19373 (yourls:1.7.3-apache)
Tagging yourls:1.7.3-apache
Tagging yourls:1.7-apache
Tagging yourls:1-apache
Tagging yourls:apache
Tagging yourls:1.7.3
Tagging yourls:1.7
Tagging yourls:1
Tagging yourls:latest

$ test/run.sh yourls:1.7.3-apache
testing yourls:1.7.3-apache
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build yourls:1.7.3-fpm
Using bashbrew/cache:fc2a04126b8e55229df81bd56ea64fec40723a9efae21aff7825198f56621274 (yourls:1.7.3-fpm)
Tagging yourls:1.7.3-fpm
Tagging yourls:1.7-fpm
Tagging yourls:1-fpm
Tagging yourls:fpm

$ test/run.sh yourls:1.7.3-fpm
testing yourls:1.7.3-fpm
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build yourls:1.7.3-fpm-alpine
Using bashbrew/cache:e064c41bac66e2959cbcc2d2b176b0b993ad8dbdc5ba5b2d03da93da6af12037 (yourls:1.7.3-fpm-alpine)
Tagging yourls:1.7.3-fpm-alpine
Tagging yourls:1.7-fpm-alpine
Tagging yourls:1-fpm-alpine
Tagging yourls:fpm-alpine

$ test/run.sh yourls:1.7.3-fpm-alpine
testing yourls:1.7.3-fpm-alpine
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed

```

**Pull request merged**